.github/workflows/tests.yml

```diff
@@ -10,23 +10,44 @@ on:
     branches: [ main ]
 
 jobs:
-  build:
+  lsst-tests:
+    name: tests
+    strategy:
+      matrix:
+        pyver: [3.8]
 
     runs-on: ubuntu-latest
 
     steps:
+    - name: Cancel Previous Runs
+      uses: styfle/cancel-workflow-action@0.9.0
+      with:
+        access_token: ${{ github.token }}
+
     - uses: actions/checkout@v2
-    - name: Set up Python 3.8
-      uses: actions/setup-python@v2
+
+    - uses: conda-incubator/setup-miniconda@v2
       with:
-        python-version: "3.8"
+          python-version: ${{ matrix.pyver }}
+          channels: conda-forge,defaults
+          channel-priority: strict
+          show-channel-urls: true
+
     - name: Install dependencies
+      shell: bash -l {0}
       run: |
+        conda config --set always_yes yes
+        conda install --quiet --yes mamba
+        mamba install --quiet --yes --file requirements.txt
+        mamba install --quiet --yes --file requirements_dev.txt
+        mamba install -q -y stackvana=0 lsstdesc-weaklensingdeblending
+      
         python -m pip install --upgrade pip
         python -m pip install --upgrade pytest
         python -m pip install flake8 pytest pytest-cov coveralls
         if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
-        if [ -f requirements.txt ]; then pip install -r test_requirements.txt; fi
+        if [ -f requirements.txt ]; then pip install -r test_requirements.txt;
```

Comment:
here you need to change the condition for the file name to test_requirements.txt as well